erts/emulator/test/persistent_term_SUITE.erl

```diff
@@ -0,0 +1,415 @@
+%%
+%% %CopyrightBegin%
+%%
+%% Copyright Ericsson AB 2017. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%5
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%
+%% %CopyrightEnd%
+%%
+
+-module(persistent_term_SUITE).
+-include_lib("common_test/include/ct.hrl").
+
+-export([all/0,suite/0,
+	 basic/1,purging/1,sharing/1,get_trapping/1,
+         info/1,info_trapping/1,off_heap_values/1,keys/1]).
+
+suite() ->
+    [{ct_hooks,[ts_install_cth]},
+     {timetrap,{minutes,10}}].
+
+all() ->
+    [basic,purging,sharing,get_trapping,info,info_trapping,
+     off_heap_values,keys].
+
+basic(_Config) ->
```

Comment:
Do these tests trigger many hash collisions? Maybe use something like what the maps tests does to generate terms that collide and put them into the hash?

Reply:
Yes, I am pretty sure that there are collisions. But just in case, I will add a test case that uses keys with known collisions.